docs/maintaining/tasks.md

```diff
@@ -14,13 +14,16 @@ To build new images and publish them to the Docker Hub registry, do the followin
 
 ## Updating the Ubuntu Base Image
 
-When there's a security fix in the Ubuntu base image or after some time passes, it's a good idea to update the pinned SHA in the
-[jupyter/base-notebook Dockerfile](https://github.com/jupyter/docker-stacks/blob/master/base-notebook/Dockerfile).
-Submit it as a regular PR and go through the build process.
-Expect the build to take a while to complete: every image layer will rebuild.
+Latest LTS Ubuntu docker image is used as the base image for `minimal-notebook`.
+We rebuild our images automatically each week, which means they receive the updates quite frequently.
+
+When there's a security fix in the Ubuntu base image, it's a good idea to manually trigger images rebuild [here](https://github.com/jupyter/docker-stacks/actions/workflows/docker.yml).
+Pushing `Run Workflow` button will trigger this process.
 
 ## Adding a New Core Image to Docker Hub
 
+_Note: in general, we do not add new core images and ask contributors to either create a [recipe](../using/recipes.md) or [community stack](../using/stacks.md)_
```

Comment:
````suggestion
```{note}
In general, we do not add new core images and ask contributors to either create a [recipe](../using/recipes.md) or [community stack](../using/stacks.md).
```
````